.travis.yml

```diff
@@ -24,9 +23,9 @@ addons:
 env:
     global:
         # The following versions are the 'default' for tests, unless
-        # overidden underneath. They are defined here in order to save having
+        # overridden underneath. They are defined here in order to save having
         # to repeat them for all configurations.
-        - NUMPY_VERSION=1.9
+        - NUMPY_VERSION=stable
```

Comment:
Your example uses an old (pre ci-helpers) and incomplete version of the package template. With that one the meaning of ``stable`` needed to be defined in the ``.travis.yml``, e.g:

```
if $NUMPY_VERSION == stable; then conda install numpy; fi
```

I would suggest to use the version from this PR as your travis template.